### PR TITLE
Rustdocs

### PR DESCRIPTION
**Pull request by gruberb**

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - name: Create Index file
+        run: ./ci/create_index_for_rust_docs.sh
+
+      - name: Deploy Docs
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_branch: gh-pages
+          publish_dir: ./target/doc
+          cname: reference.centrifuge.io
```
Comment (gruberb) [inline .github/workflows/docs.yml:36]: I don't actually know why the PR says "two new files". I added  the `cname` parameter.

Comment (gruberb) [reply .github/workflows/docs.yml:36]: Ah, I should have opened a new branch. This shows the changes from last time as well. Hope we can still merge it like that?

Comment (NunoAlexandre) [reply .github/workflows/docs.yml:36]: Maybe the base of this PR is not correct?

Comment (NunoAlexandre) [reply .github/workflows/docs.yml:36]: Yeah it seems so

Comment (gruberb) [reply .github/workflows/docs.yml:36]: Ahh of course, thank you!

**Review by offerijns — APPROVED** [review]

**Pull request merged**